components/service/telemetry/src/main/java/org/mozilla/telemetry/measurement/MetricsMeasurement.java

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.telemetry.measurement;
+
+import android.content.SharedPreferences;
+import android.support.annotation.VisibleForTesting;
+import android.util.Log;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.mozilla.telemetry.config.TelemetryConfiguration;
+import org.mozilla.telemetry.metrics.TelemetryMetrics;
+import org.mozilla.telemetry.util.IOUtils;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+
+public class MetricsMeasurement extends TelemetryMeasurement {
```

Comment:
Metrics will work differently. They do not need to be saved to disk. They will be added at the time we create the ping.